src/server/rest/v1/service/security/StatisticSecurity.ts

```diff
@@ -10,7 +10,7 @@ export default class StatisticSecurity {
       StartDateTime: request.StartDateTime ? sanitize(request.StartDateTime) : null,
       EndDateTime: request.EndDateTime ? sanitize(request.EndDateTime) : null,
       SiteAreaIDs: request.SiteAreaID ? sanitize(request.SiteAreaID).split('|') : null,
-      ChargeBoxIDs: request.ChargeBoxID ? sanitize(request.ChargeBoxID).split('|') : null,
+      ChargingStationIDs: request.ChargingStationID ? sanitize(request.ChargingStationID).split('|') : null,
```

Comment:
All the splits should be done outside the Security class (can be done when schema will be implemented)